inference-engine/src/mkldnn_plugin/nodes/proposal_imp.cpp

```diff
@@ -322,6 +323,10 @@ void proposal_exec(const float* input0, const float* input1,
     const float img_H = img_info[conf.swap_xy ? 1 : 0];
     const float img_W = img_info[conf.swap_xy ? 0 : 1];
 
+    if (conf.normalize_ && (img_H == 0.f || img_W == 0.f)) {
```

Comment:
Lets check img_info values unconditionally for all modes of proposal usage. In other words remove `conf.normalize_` from check.

Moreover we have to check on NaN, inf, and negative values. I suggest to use `std::normal(img_H) && img_H > 0.0`.
 

Reply:
Fixed